### DISCUSSION TITLE
Package name format of go modules was changed?

### DISCUSSION THREAD

**Discussion by kamatama41** [renovatebot/renovate#8683] [Request Help] [answered]
I'm using Renovate on GitHub app.

Previously (at least until last weekend) package name format of go modules was `<domain>/<organization>/<module>`
like `github.com/aws/aws-sdk-go`.  However, it seems to be changed to just `<organization>/<module>` like `aws/aws-sdk-go`, so that our `packageRules` has not been working.

Is this a bug, or an intentional change? I couldn't find any release of the change.

This is an example of renovate.json. Automerge of aws-sdk-go is enabled again by changing matchPackagePatterns to `"aws/aws-sdk-go"`
```json
{
  "extends": [
    "config:base"
  ],
  "postUpdateOptions": [
    "gomodTidy"
  ],
  "packageRules": [
    {
      "matchPackagePatterns": [
        "^github.com/aws/aws-sdk-go"
      ],
      "automerge": true
    }
  ]
}
```

**Comment by kamatama41** [comment, marked as answer]
I checked some other package names between before and after the change and its result is following.
Especially `gopkg.in/yaml.v2` is changed to just `yaml`, which is very hard to guess new package name.

| before                       | after                  |
|------------------------------|------------------------|
| github.com/aws/aws-sdk-go    | aws/aws-sdk-go         |
| github.com/go-redis/redis/v8 | go-redis/redis/v8      |
| gopkg.in/yaml.v2             | yaml                   |
| golang.org/x/lint            | golang.org/x/lint      |
| go.uber.org/zap              | go.uber.org/zap        |
| google.golang.org/grpc       | google.golang.org/grpc |

**Reply by rarkins** [reply]
@viceice an undesired effect of depNameShort refactoring? I think we need to revert it

**Reply by viceice** [reply]
Looks like. Maybe we shouldn't shorten go modules at all instead of simply revert.

**Reply by rarkins** [reply]
Sure, let's start by not shortening

**Reply by kamatama41** [reply]
I have confirmed it's fixed in our repositories, thanks!
